src/Symfony/Component/VarExporter/Tests/VarExporterTest.php

```diff
@@ -127,9 +129,15 @@ public function provideExport()
         yield ['bool', true, true];
         yield ['simple-array', [123, ['abc']], true];
         yield ['partially-indexed-array', [5 => true, 1 => true, 2 => true, 6 => true], true];
-        yield ['datetime', \DateTime::createFromFormat('U', 0)];
-
-        $value = new \ArrayObject();
+        yield ['datetime', [
+            \DateTime::createFromFormat('U', 0),
+            \DateTimeImmutable::createFromFormat('U', 0),
+            new \DateTimeZone('Europe/Paris'),
+            new \DateInterval('P7D'),
+            new \DatePeriod('R4/2012-07-01T00:00:00Z/P7D'),
+        ]];
+
+        $value = \PHP_VERSION_ID >= 70406 ? new ArrayObject() : new \ArrayObject();
```

Comment:
I'm curious, what's the reason for this version check?

Reply:
PHP 8.2 needs the attribute for this test but I also want to keep a test that creates a native ArrayObject.

Reply:
Oh, I see. That version number 7.4.6 seemed so oddly specific that I though I had missed something in PHP's ChangeLog. 😅